packages/cardhost/app/routes/application.ts

```diff
@@ -0,0 +1,5 @@
+import Route from '@ember/routing/route';
+
+import '@cardstack/boxel/styles/boxel';
```

Comment:
I would like to investigate how to keep these concerns at the component level instead.

Reply:
That goes for all the font files too.